.github/workflows/coverage.yml

```
@@ -14,7 +14,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [15]
+        node-version: [18]
```

```suggestion
        node-version: ['lts/*']
```

May be worth moving this to just use lts, so it is one less this that needs updating?

---

Honestly, I didn't know this is a thing